src/pylorax/__init__.py

```diff
@@ -369,6 +369,21 @@ def run(self, dbo, product, version, release, variant="", bugurl="",
         logger.info("anaconda args = %s", anaconda_args)
         treebuilder.rebuild_initrds(add_args=anaconda_args)
 
+        if doupgrade:
+            # Build upgrade.img. It'd be nice if these could coexist in the same
+            # image, but that would increase the size of the anaconda initramfs,
+            # which worries some people (esp. PPC tftpboot). So they're separate.
+            try:
+                # If possible, use the 'redhat-upgrade-tool' plymouth theme
+                themes = runcmd_output(['plymouth-set-default-theme', '--list'],
```

Comment:
That's not going to do what you want it to do. runcmd runs in the context of the host system doing the build, not the packages being installed. You need to find a better way to accomplish this.

Reply:
Actually I don’t even know what this does, this is restored code from the rhel7 branch
I will have to give this a try anyway and need to get also the system upgrade dracut module back.

I should maybe haven’t already created the PR it was probably too early

Reply:
That's probably a good idea. Hack up your local copy of lorax and make sure things are all working. Add tests if you can, then do a new PR. We also need to make sure that whatever new dracut modules are needed are already available before we merge this.